export/delimited-zip/src/export.delimited.zip.plugin.ts

```
@@ -0,0 +1,201 @@
+        console.log('Sheets retrieved:', sheets)
+      }
+
+      // Get current date-time
+      const dateTime = new Date().toISOString().replace(/[:.]/g, '-')
+
+      // Get the path to the system's temporary directory
+      const tempDir = tmpdir()
+
+      const sanitizeFileName = (name: string) =>
+        path.basename(
+          name.replace(/[<>:"/\\|?*\x00-\x1F]/g, '').replace(/\s+/g, '_')
```

Biome

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)